miio/airqualitymonitor.py

```diff
@@ -232,7 +245,23 @@ def status(self) -> AirQualityMonitorStatus:
 
         properties = AVAILABLE_PROPERTIES[self.model]
 
-        values = self.send("get_prop", properties)
+        if self.model != "cgllc.airmonitor.b1":
+            values = self.send("get_prop", properties)
+        else:
+            values = self.send("get_air_data")
+            """
+            Format like on the display of the unit
+            In the current situation tvoc_unit and temperature_unit does not translate the tvoc/temperature values
+            """
+            values['co2']              = round(values['co2e'],1)
+            # Replace key co2e by just co2
+            values.pop('co2e', None)
+            values['humidity']         = round(values['humidity'],1)
```

Comment:
Please don't manipulate the measurements. Pass all measurements unchanged to a new status class (cp. FanStatusP5 of fan.py).